meta/main.yml

```diff
@@ -3,6 +3,8 @@ galaxy_info:
   author: Mark Feldhousen
```

Comment:
Should author and company get updated?

Reply:
Mark is still the original author.  Why would the company be updated?